test/metabase/query_processor/util/add_alias_info_test.clj

```diff
@@ -856,23 +859,25 @@
                                     :native   {:template-tags {} :query "select 1 as b1, 2 as b2;"}}
                   :result-metadata [(result-metadata-for eid "B1")
                                     (result-metadata-for eid "B2")]})
-               {:name            "Joined"
-                :id              3
-                :database-id     (meta/id)
-                :type            :model
-                :dataset-query   {:database (meta/id)
-                                  :type     :query
-                                  :query    {:joins
-                                             [{:fields :all,
-                                               :alias "Model B - A1",
-                                               :ident "t3Nz_yY5hISIlmxaJlSsm"
-                                               :strategy :inner-join,
-                                               :condition
-                                               [:=
-                                                [:field "A1" {:base-type :type/Integer}]
-                                                [:field "B1" {:base-type :type/Integer, :join-alias "Model B - A1"}]],
-                                               :source-table "card__2"}],
-                                             :source-table "card__1"}}}]}))))
+               (let [eid (u/generate-nano-id)]
```

Comment:
Is this supposed to be reused somewhere?

Reply:
I think I was just following the shape of the ones above, but yeah, it's not reused.